ibis/expr/operations.py

```diff
@@ -80,40 +322,25 @@ def blocks(self):
 
 
 class UnboundTable(PhysicalTable):
-
-    def __init__(self, schema, name=None):
-        if name is None:
-            name = genname()
-        TableNode.__init__(self, [schema, name])
-        HasSchema.__init__(self, schema, name=name)
+    schema = rlz.schema
+    name = rlz.optional(rlz.instanceof(six.string_types), default=genname)
 
 
 class DatabaseTable(PhysicalTable):
-
-    """
-
-    """
-
-    def __init__(self, name, schema, source):
-        self.source = source
-
-        TableNode.__init__(self, [name, schema, source])
-        HasSchema.__init__(self, schema, name=name)
+    name = rlz.instanceof(six.string_types)
+    schema = rlz.schema
+    source = rlz.noop
```

Comment:
These `noop`s should be instances of `ibis.client.Client`, right?

Reply:
Yes, there are a lot of `noop`s currently which we should eventually configure.